app/models/batch_arbitrary_assignment_creation.rb

```diff
@@ -0,0 +1,25 @@
+class BatchArbitraryAssignmentCreation
+  attr_reader :visitor_id, :assignments, :force
+
+  def initialize(
+    visitor_id: nil,
+    assignments: nil,
```

Comment:
```suggestion
    visitor_id:,
    assignments:,
```

Any reason to leave these as optional?

Reply:
nope, i copied the `ArbitraryAssignmentCreation`, but it doesn't make sense to me that they're optional!